### DISCUSSION TITLE
Training LayoutLM on wildreceipt dataset

### DISCUSSION THREAD

**Discussion by riyajatar93** [PaddlePaddle/PaddleOCR#12856] [Q&A]
Can I train vi_layoutlm on wildreceipt dataset?
If yes , what needs to changed in its config or dataset preparation stage?

Thanks

**Comment by EphronM** [comment]
@riyajatar93  i have the same requirement. have you found any solution??

**Comment by UserWangZz** [comment]
参考文档https://paddlepaddle.github.io/PaddleOCR/ppstructure/blog/how_to_do_kie.html#221-ser
